config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/iam-controller
-  newTag: 1.2.6
+  newTag: 1.2.5
```

Comment:
I pulled the latest iam-controller changes that changes the tag to 1.2.6 and rebased my local branch:
- https://github.com/kunhwiko/ack-iam-controller

After running code generate, it seems to be setting this to 1.2.5. Is there something I might have missed?

Reply:
You need to run `git fetch upstream --tags` with upstream pointing to `git@github.com:aws-controllers-k8s/iam-controller`